_topic_maps/_topic_map.yml

```diff
@@ -2791,6 +2791,15 @@ Topics:
       File: backing-up-applications
     - Name: Restoring applications
       File: restoring-applications
+  - Name: OADP and ROSA 
+    Dir: oadp-rosa
+    Topics:
+      - Name: Backing up applications ROSA clusters using OADP
```

Comment:
The ROSA cluster configuration is mentioned in below pages is actually a ROSA with STS cluster. I would prefer to mention the STS as well. Maybe something like .

**Application Backup on ROSA STS Using OADP**